contrib/compare_ids_test.py

```diff
@@ -11,21 +13,31 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+import sys
+import os
```

Comment:
os and sys are not used. Can you run lint on all the py files

Reply:
Yup. Did pyformat, forgot lint. Fixed.